virttest/qemu_devices/qcontainer.py

```diff
@@ -3008,13 +3009,18 @@ def fs_define_by_params(self, name, params, bus=None):
 
         devices = []
         if driver == "virtio-fs":
-            binary = params.get("fs_binary", "/usr/libexec/virtiofsd")
-            extra_options = params.get("fs_binary_extra_options")
-            enable_debug_mode = params.get('fs_enable_debug_mode', 'no') == "yes"
-            sock_path = os.path.join(data_dir.get_tmp_dir(),
-                                     '-'.join((self.vmname, name, 'virtiofsd.sock')))
-            vfsd = qdevices.QVirtioFSDev(name, binary, sock_path,
-                                         source, extra_options, enable_debug_mode)
+            if fs_source_user_config == "yes":
+                sock_path = params.get("fs_source_user_config_sock_path")
```

Comment:
```suggestion
                sock_path = params.get("fs_source_user_sock_path")
```
The word `config` may be redundant.

Reply:
Thanks for the reminder, corrected.